### ISSUE TITLE
Signature Match issue.

### ISSUE THREAD

**Issue by Tejas-narola** [Kickflip/kickflip-ios-sdk#31]
We are getting signature match issue for amazon s3. Would you please guide us?

**Comment by chrisballinger** [comment]
Using your own S3 keys (not via the Kickflip.io website) is unsupported.

**Comment by Tejas-narola** [comment]
Yes we are using s3 keys via "https://kickflip.io/api/1.2" only. would you please guide us why it is still creating issue?

**Comment by Tejas-narola** [comment]
We are getting this error when s3 is trying to put object .

"Message=The request signature we calculated does not match the signature you provided. Check your key and signing method"

Could you please help us?
